### PR TITLE
More icons

### PR DESCRIPTION
**Pull request by biscuitWizard**
Credits to Eris, /tg/, and yogstation for their icons.

### CODE REVIEW

File: code/modules/client/preference_setup/loadout/lists/utility.dm [hunk 1]
```diff
@@ -26,7 +26,7 @@
 		"blue folder" =   /obj/item/folder/blue,
 		"grey folder" =   /obj/item/folder,
 		"red folder" =    /obj/item/folder/red,
-		"white folder" =  /obj/item/folder/white,
+		"white folder" =  /obj/item/folder/cyan,
```
Comment (comma) [inline code/modules/client/preference_setup/loadout/lists/utility.dm]: change name too

**Comment by comma** [comment]
beakers are kinda too opaque but maybe glass alpha will do the trick
set `	applies_material_colour = TRUE` on beaker definition (and to false on weird subtypes like cryo/bluespace)

missing `beakerlarge25` reagent filling icon

not really a fan of new scanner icons not being obvious but nothing that can't be adjusted later.

**Comment by Techhead0** [comment]
Personal opinions but putting them here so they're in one place
Like most of the icons, don't like the following new ones: Body bag, spectrometers, beakers, plant scanner, ore box, folders, portable scrubbers

Non-personal stuff: The janicart and scrubbers have overlay sprites that were not brought over.

**Comment by biscuitWizard** [comment]
@Techhead0 That was intentional re: janitor carts. Overlay sprites would require code and I don't feel like bringing them over /w code. If someone else wants to in a future PR, they are from Eris.

**Comment by comma** [comment]
Janicart already applies overlays tho
```
/obj/structure/janitorialcart/on_update_icon()
	overlays.Cut()
	if(mybag)
		overlays += "cart_garbage"
	if(mymop)
		overlays += "cart_mop"
	if(myspray)
		overlays += "cart_spray"
	if(myreplacer)
		overlays += "cart_replacer"
	if(signs)
		overlays += "cart_sign[signs]"
```

**Comment by biscuitWizard** [comment]
All comments addressed.

**Review by comma — APPROVED** [review]

**Pull request merged**